Devscord.DiscordFramework/Workflow.cs

```diff
@@ -116,7 +117,7 @@ private void RunWithDiscordCommandMethods(IController controller, DiscordRequest
                 {
                     if (method.HasAttribute<AdminCommand>() && !contexts.User.IsAdmin)
                     {
-                        var messageService = new MessagesServiceFactory(new ResponsesService()).Create(contexts);
+                        var messageService = new MessagesServiceFactory(new ResponsesService(), new MessageSplittingService()).Create(contexts);
```

Comment:
fajnie byłoby to wyciągać z autofaca, ale jak sie nie da (bez większej przebudowy) to zostaw, później sie naprawi

Reply:
myślę że dodanie autofaca do frameworka mogłoby być jako osobny task